### PR TITLE
added command "make lint"

### PR DESCRIPTION
**Pull request by miiu96**
Added in the Makefile command "make lint" that will run` golangci linter.`
This command will check if the `golangci linter` is installed ( in case of not installed, will install the latest tag of the `golangci linter`) and after that will run the linter.

### CODE REVIEW

**Review by iulianpascalau — DISMISSED** [review]

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
works only for the first issue right now

**Review by bogdan-rosianu — APPROVED** [review]

File: crypto/signing/multisig/pubkeys.go [hunk 1]
```diff
@@ -5,10 +5,9 @@ import (
 )
 
 func convertStringsToPubKeys(pubKeys []string, kg crypto.KeyGenerator) ([]crypto.PublicKey, error) {
-	var pk []crypto.PublicKey
-
-	//convert pubKeys
-	for _, pubKeyStr := range pubKeys {
+	// convert pubKeys
+	pk := make([]crypto.PublicKey, len(pubKeys))
```
Comment (iulianpascalau) [inline crypto/signing/multisig/pubkeys.go:9]: please use:
```GO
pk := make([]crypto.PublicKey, 0, len(pubKeys))
for _, pubKeyStr := range pubKeys {
		if pubKeyStr == "" {
			return nil, crypto.ErrEmptyPubKeyString
		}
	        //................
		pk = append(pk, pubKey)
}
```

**Review by iulianpascalau — CHANGES_REQUESTED** [review]

File: crypto/signing/multisig/pubkeys.go (hunk 1 above)
Comment (miiu96) [reply crypto/signing/multisig/pubkeys.go]: done.

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**